Pinta.Gui.Widgets/Dialogs/SimpleEffectDialog.cs

```diff
@@ -398,12 +391,14 @@ private static Gtk.Label CreateHintLabel (string hint)
 		return label;
 	}
 
+	private readonly Random random = new ();
```

Comment:
I'd prefer having this moved up back to the top, so that all member variables are declared together.

Reply:
No problem, I just moved it to the top of the class